### PR TITLE
Do not panic inside the lib, handle errors in main()

### PR DESCRIPTION
**Pull request by svenfoo**

### CODE REVIEW

**Comment by svenfoo** [comment]
As far as I can see this is entirely obsolete now that the argument parser has been merged. For now I'll leave it open for review anyway, and have another look at it later.

File: src/lib.rs [hunk 1]
```diff
@@ -13,13 +13,11 @@ pub mod models;
...
-    let resume: Resume = serde_yaml::from_str(&contents).unwrap();
-    let mut file = File::create("output.tex")?;
-    match file.write_all(&resume.render().unwrap().as_bytes()) {
-        Err(why) => panic!("Couldn't write to disk: {}", why.description()),
-        Ok(_) => println!("Done!"),
-    }
-    Ok(())
+    let resume: Resume = serde_yaml::from_str(&contents)?;
+    let mut file = File::create(output)?;
+    file.write_all(&resume.render()?.as_bytes())?;
```
Comment (dlalic) [inline src/lib.rs]: Nice.

**Comment by svenfoo** [comment]
Rewritten based on current master.

File: src/lib.rs [hunk 2]
```diff
@@ -8,13 +7,14 @@ mod integrations;
...
-        println!("Please specify a subcommand. Use --help for help.");
+pub enum Integration {
+    Tex,
+    Xing
+}
+
+pub fn run(int : Integration, config: &str) -> Result<String, Box<dyn Error>> {
+    match int {
+        Integration::Tex => execute(config),
+        Integration::Xing => panic!("not yet implemented")
```
Comment (dlalic) [inline src/lib.rs]: Neat.

Comment (dlalic) [reply src/lib.rs]: There's [`unimplemented!();`](https://doc.rust-lang.org/std/macro.unimplemented.html) as a similar option.

**Pull request merged**